python-package/README.rst

```diff
@@ -204,6 +204,10 @@ You can use ``python setup.py bdist_wheel`` instead of ``python setup.py install
 Install Dask-package
 ''''''''''''''''''''
 
+.. warning::
+
+    Dask-package is only tested on Linux.
```

Comment:
When this file is viewed on GitHub, this warning doesn't render the way it would in Sphinx.

![image](https://user-images.githubusercontent.com/7608904/111081035-5428f880-84cf-11eb-9974-fbbba73e8ea1.png)

Are you ok with that? It doesn't bother me because I hope we'll be able to remove this soon anyway, but just wanted you to be aware.

Reply:
Yeah, like all other `Warning`s and `Note`s in other RSTs:

![image](https://user-images.githubusercontent.com/25141164/111081209-41381780-8513-11eb-83d1-86fddfbf401b.png)

I believe there should be no problem with rendering on PyPI because http://rst.ninjs.org/ shows no errors when I paste raw content of `python-package/README.rst`.
FYI, PyPI used to show blank page in case of any errors in RST formatting of uploaded README, but I don't know what's the current policy of error handling there.

Reply:
ok sounds good!